### DISCUSSION TITLE
Urgent! After tonight's API maintenance, my store is put down. Custom fields for products are not fetched.

### DISCUSSION THREAD

**Discussion by agsmartwolf** [swellstores/community#198] [Help] [answered]
I have a 400 response error for graphql request after tonight's API maintenance.
<img width="1070" alt="Screenshot 2023-04-06 at 13 04 27" src="https://user-images.githubusercontent.com/109105178/230329425-daff760a-80dd-429c-95bf-1a905c256bb4.png">

**Comment by agsmartwolf** [comment]
I have permissions. Everything worked and i didn't have any new deployments.
This error has started to appear after Swell's tonight work.
<img width="628" alt="image" src="https://user-images.githubusercontent.com/109105178/230330363-3ccda73b-7ae3-43e1-9a27-23ca36b0c339.png">

**Comment by agsmartwolf** [comment]
It's very urgent guys!
We launched an AD tonight, and now the store doesn't work.

**Comment by rodrigoddalmeida** [comment]
@agsmartwolf could you try again? Please feel free to use `Cache-control: no-cache` as a way to refresh the schema generated for your API.

**Reply by agsmartwolf** [reply]
It's a 200 status now. Bit there is next error:

<img width="896" alt="Screenshot 2023-04-06 at 14 18 45" src="https://user-images.githubusercontent.com/109105178/230348858-9185a708-91fc-4a40-9ace-a665358c73fb.png">

**Reply by agsmartwolf** [reply]
I've tried to launch a local server and now i have the next error when i try to request store's settings:
<img width="979" alt="Screenshot 2023-04-06 at 14 37 35" src="https://user-images.githubusercontent.com/109105178/230353373-5388b970-493d-4f95-8bbb-b8dc97426572.png">

**Comment by agsmartwolf** [comment]
I had the last deployment 2 days ago. So it has been working for 2 days in production, and the schema hasn't changed. 
I haven't updated anything. The only thing that happened was tonight's maintenance.

So I got reports this morning about unavailable catalog and product pages today.

**Comment by rodrigoddalmeida** [comment, marked as answer]
@agsmartwolf we have recovered our service levels, let us know if you have the same issues.

**Reply by agsmartwolf** [reply]
It's working right now. Thank you for resolving this.
